<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>环境设置工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>一键优化</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>设备基本信息</string>
    </property>
    <widget class="QLabel" name="label_hardware_CPUmodel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU型号    ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_CPUnum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU核心数  ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_GPUmodel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显卡型号   ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_GPUmemory">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显卡显存   ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_CPUspeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU主频    ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_Directx10">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DirectX 10 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_vc2005x64">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VC2005 x64 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_Directx11">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DirectX 11 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_setup1">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Setup1     ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_memory">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>内存上限   ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_CPUmodel_result">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_CPUnum_result">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_CPUspeed_result">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_GPUmodel_result">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_memory_result">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware_GPUmemory_result">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_vc2005x64_result">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_Directx11_result">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_Directx10_result">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software_setup1_result">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hardware">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>硬件：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_software">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>软件：</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_usable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前系统是否可以使用非编软件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optimize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前系统参数与推荐值是否匹配：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_usable_result">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>可以使用</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optimize_result">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>无需优化</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>optimizeOneKey()</slot>
 </slots>
</ui>
